tiledb/sm/query/readers/result_tile.h

```diff
@@ -240,6 +264,31 @@ class ResultTile {
       return validity_filtered_data_;
     }
 
+    /** @return The fixed filtered data I/O task. */
+    inline ThreadPool::SharedTask fixed_filtered_data_task() const {
+      return fixed_filtered_data_task_;
+    }
+
+    /** @return The var filtered data I/O task. */
+    inline ThreadPool::SharedTask var_filtered_data_task() const {
+      return var_filtered_data_task_;
+    }
+
+    /** @return The validity filtered data I/O task. */
+    inline ThreadPool::SharedTask validity_filtered_data_task() const {
+      return validity_filtered_data_task_;
+    }
+
+    /** @return shared_ptr to FilteredData block used by this Tile. */
+    inline shared_ptr<FilteredData> filtered_data() const {
+      return filtered_data_;
+    }
+
+    /** Clear the held filtered data. */
+    inline void clear_filtered_data() {
```

Comment:
I'd suggest `release` for nomenclature since `clear` could be confused for a memory write.